src/crypto/hsm/nxp/CHIPCryptoPALHsm_SE05X_P256.cpp

```diff
@@ -202,11 +208,12 @@ CHIP_ERROR P256KeypairHSM::ECDSA_sign_msg(const uint8_t * msg, size_t msg_length
 
 CHIP_ERROR P256KeypairHSM::ECDSA_sign_hash(const uint8_t * hash, size_t hash_length, P256ECDSASignature & out_signature)
 {
-    CHIP_ERROR error           = CHIP_ERROR_INTERNAL;
-    sss_asymmetric_t asymm_ctx = { 0 };
-    sss_status_t status        = kStatus_SSS_Success;
-    sss_object_t keyObject     = { 0 };
-    size_t siglen              = out_signature.Capacity();
+    CHIP_ERROR error            = CHIP_ERROR_INTERNAL;
+    sss_asymmetric_t asymm_ctx  = { 0 };
+    sss_status_t status         = kStatus_SSS_Success;
+    sss_object_t keyObject      = { 0 };
+    uint8_t signature_se05x[80] = { 0 };
```

Comment:
```suggestion
    uint8_t signature_se05x[kMax_ECDSA_Signature_Length_Der] = { 0 };
```

Reply:
Why is it 80? A P256 signature in DER will always fit in `kMax_ECDSA_Signature_Length_Der`. I can understand keys needing more bytes, but for P256, 80 is not possible in DER.